xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1202,10 +1202,10 @@ bool CDVDDemuxFFmpeg::SeekTime(double time, bool backwards, double* startpts)
     if (ret < 0)
     {
       int64_t starttime = m_pFormatContext->start_time;
-      if (m_checkvideo)
+      if (m_checkTransportStream)
       {
         AVStream* st = m_pFormatContext->streams[m_seekStream];
-        starttime = av_rescale(m_startTime, st->time_base.num, st->time_base.den);
+        starttime = av_rescale(static_cast<int64_t>(m_startTime), st->time_base.num, st->time_base.den);
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
Fixed. Column width was exactly 100 which is why I left it that way.

Reply:
fixed

Reply:
I got the .clang-format file to work for me finally and reformatted all the affected areas.  I don't think you'll like that it put 4 spaces instead of 2 in one place, but that's what it did so I left it alone